introduction/silo-entry-state.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Silo Entry State"
+icon: "key"
+description: "Understanding Silo Entry States in Invopop Workflows"
+---
+
+Let's take a step back and look at the full picture.
+
+The service we use to store GOBL documents is called **Silo**. Hence, every document uploaded to this service is referred to as a **Silo Entry**.
+
+Each silo entry follows a JSON schema with multiple fields. You can find a full breakdown of these fields in the Invopop API Reference under [Fetch an entry](/api-ref/silo/entries/fetch-an-entry).
+
+One of these fields is the state field, which we refer to as the **Silo Entry State**. Invopop provides a predefined set of values for this field to help standardize usage across workflows. These values reflect the current stage of a document within your workflow.
+
+The available values include:
+	1.	`Draft`: Document created but not yet in a workflow.
+	2.	`Processing`: Document is being prepared or transformed.
+	3.	`Registered`: Document marked as registered in your system.
+	4.	`Sent`: Document marked as sent in your system.
+	5.	`Paid`: Document marked as paid by your system.
+	6.	`Error`: A processing or submission step failed.
+	7.	`Void`: Document canceled or invalidated.
+
+Silo entry states are user-defined and used for internal tracking only. They do not represent confirmations of external events such as successful registration with a tax authority or delivery to an endpoint.
+
+For example, when the silo entry state is set to `Sent`, it means the document has been flagged as submitted within your workflow, but it does not confirm that the tax agency has received or validated it.
```

Comment:
Technically, it can mean whatever the user wants it to mean. I'd suggest using language like "Setting the state to 'sent' in your workflow, could be used to imply that the document has been successfully sent to a tax agency."